### PR TITLE
[grocy] Add Grocy

### PR DESCRIPTION
**Pull request by carpenike**
#### Special notes for your reviewer:
New chart supporting Grocy.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/grocy/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: grocy
+version: 1.0.0
+keywords:
+  - grocy
+home: https://github.com/billimek/billimek-charts/tree/master/charts/grocy
+icon: https://github.com/grocy/grocy/raw/master/public/img/appicons/mstile-150x150.png
+sources:
+  - https://github.com/grocy/grocy
+maintainers:
+  - name: Ryan Holt
```
Comment (billimek) [inline charts/grocy/Chart.yaml]: Can you please make this your github username (`carpenike`)? The linter seems to only want github usernames.

File: charts/grocy/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+| `strategyType`             | Specifies the strategy used to replace old Pods by new ones | `Recreate` |
+| `timezone`                 | Timezone the grocy instance should run as, e.g. 'America/New_York' | `UTC` |
+| `puid`                     | process userID the grocy instance should run as | `1000` |
+| `pgid`                     | process groupID the grocy instance should run as | `1000` |
+<!-- | `probes.liveness.initialDelaySeconds`  | Specify liveness `initialDelaySeconds` parameter for the deployment  | `60` |
+| `probes.liveness.failureThreshold`     | Specify liveness `failureThreshold` parameter for the deployment     | `5`  |
+| `probes.liveness.timeoutSeconds`       | Specify liveness `timeoutSeconds` parameter for the deployment       | `10` |
+| `probes.readiness.initialDelaySeconds` | Specify readiness `initialDelaySeconds` parameter for the deployment | `60` |
+| `probes.readiness.failureThreshold`    | Specify readiness `failureThreshold` parameter for the deployment    | `5`  |
+| `probes.readiness.timeoutSeconds`      | Specify readiness `timeoutSeconds` parameter for the deployment      | `10` | -->
```
Comment (billimek) [inline charts/grocy/README.md]: Should these be commented??

File: charts/grocy/Chart.yaml (hunk 1 above)
Comment (carpenike) [reply charts/grocy/Chart.yaml]: resolved by 537ff35

File: charts/grocy/README.md (hunk 2 above)
Comment (carpenike) [reply charts/grocy/README.md]: woops... should be resolved by 537ff35

**Pull request merged**